rhizome/common/lib/util.rb

```diff
@@ -70,3 +70,7 @@ def safe_write_to_file(filename, content = nil)
     File.rename(temp_filename, filename)
   end
 end
+
+def curl_file(url, path)
```

Comment:
We might also use this helper in the `BootImage` class in the future

Reply:
yes, there should be de-duplication at some point.